browse-scaler/src/index.js

```diff
@@ -3,8 +3,8 @@ const { getCollectionLevelBrowseImage, getGranuleLevelBrowseImage } = require('.
 const { cacheImage, getImageFromCache } = require('./cache');
 const { withTimeout, slurpImageIntoBuffer } = require('./util');
 
-const timeoutInterval = process.env.EXTERNAL_REQUEST_TIMEOUT || 2000;
-
+const timeoutInterval = process.env.EXTERNAL_REQUEST_TIMEOUT || 1000;
+console.log(`TIMEOUT INTERVAL IS: ${timeoutInterval}`);
```

Comment:
I don't think we need this log message

Reply:
I think it really helps the debugging - especially after we have the environment variable. if somewhere it times out again, we will need to know why.